terraform-modules/github-runner/gce-instance.tf

```diff
@@ -0,0 +1,55 @@
+resource "google_compute_address" "static" {
+  provider = google.default
+  count    = var.runners
+
+  name = "github-runner-${var.repo}-${count.index + 1}-ip"
+}
+
+data "google_compute_image" "debian" {
+  provider = google.default
+
+  family  = "debian-10"
+  project = "debian-cloud"
+}
+
+resource "google_compute_instance" "runner" {
+  provider   = google.default
+  depends_on = [google_compute_address.static]
+  count      = var.runners
+
+  name                      = "github-runner-${var.repo}-${count.index + 1}"
+  description               = "GitHub Actions runner ${count.index + 1} for broadinstitute/${var.repo}"
+  machine_type              = var.machine-type
+  zone                      = var.zone
+  allow_stopping_for_update = true
+
+  service_account {
+    email  = var.service-account
+    scopes = ["default"]
```

Comment:
If we use this github runner for WFL integration tests we might need to change these default scopes. For instance to make the cloud function talk to WFL it needed to set these [scopes](https://github.com/broadinstitute/wfl/blob/master/cloud_function/main.py#L19).

Reply:
Good point. This isn't meant for WFL integration tests but I didn't realize how scopes worked. I've set the defaults to be what the cloud function needs (the SA will still need permissions as I understand it) but it is a variable now so we can change it per-repo.

Reply:
Oh I see! I don't think anything in `gotc-deploy` makes API requests to WFL so now I'm not entirely sure what scopes it requires. I guess we'll find out if we encounter any permissions errors!